src/main/java/com/iota/iri/network/pipeline/TransactionProcessingPipelineImpl.java

```diff
@@ -191,6 +200,23 @@ public void process(byte[] txTrits) {
         hashAndValidate(new ProcessingContext(payload));
     }
 
+    @Override
+    public void refillBroadcastQueue(){
+        try{
+            Iterator<TransactionViewModel> hashIterator = txSolidifier.getBroadcastQueue().iterator();
+            Set<TransactionViewModel> toRemove = new LinkedHashSet<>();
+            while(!Thread.currentThread().isInterrupted() && hashIterator.hasNext()){
+                TransactionViewModel t = hashIterator.next();
+                broadcastStageQueue.put(new ProcessingContext(new BroadcastPayload(null, t)));
+                toRemove.add(t);
```

Comment:
This refill is what adds solid transactions to the broadcast queue, so here, yes you would send it to all neighbours. I'll change the 1 letter variable though.

Reply:
Just making sure, now with the new stage, this problem is resolved

Reply:
Due to the asynchrounous nature of solidification, there is still no neighbour variable to pass through unless we create a mapping, which imo is not worth it to do. So anything that gets updated as solid will be broadcast to all neighbours. But now only transactions that are solid will end up in the broadcast queue. We no longer forward transactions that have just been received.

Reply:
I think we need to create this mapping
I also don't think we should touch this PR too much, so please create an issue to do this :-)

Reply:
@DyrellC I am marking this as a point to discuss before merging

Reply:
We will need to discuss this, because now that the solidify stage does not add the incoming transactions to the solidification queue anymore, we have no entry point to put the origin neighbour into the process. As is the only place that places transactions into the solidification queue is from the milestone solidifier. This is something that could be introduced in the milestone stage, I mention that in the new issue #1701